DataFormats/TrackerCommon/src/TrackerTopology.cc

```diff
@@ -5,8 +5,19 @@
 #include <sstream>
 
 TrackerTopology::TrackerTopology( const PixelBarrelValues& pxb, const PixelEndcapValues& pxf,
-				  const TECValues& tecv, const TIBValues& tibv, 
-				  const TIDValues& tidv, const TOBValues& tobv) {
+                                  const TECValues& tecv, const TIBValues& tibv, 
+                                  const TIDValues& tidv, const TOBValues& tobv) 
+  : bits_per_field{
```

Comment:
since you opened up the initializer list in this constructor, please move other data members here as well from the body of the constructor. It's a preferred style anyways.